setup/artifacts/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: single
+title:  "Configure Artifacts"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Artifacts are remote, deployable resources in Spinnaker. Depending on
```

Comment:
What do we mean by "remote", especially if we also support embedded artifacts?

Reply:
Good point. I understand what you mean by remote, but do embedded artifacts begin their lives as remote artifacts, @lwander ?